app/Screens/Auction1.js

```diff
@@ -1,18 +1,148 @@
+  const [useruni, setUseruni] = useState('');
+  
+
+  useEffect(() => {
+    (async () => {
+      if (Platform.OS !== 'web') {
+        const { status } = await ImagePicker.requestMediaLibraryPermissionsAsync();
+        if (status !== 'granted') {
+          alert('Sorry, we need camera roll permissions to make this work!');
+        }
+      }
+    })();
+  }, []);
+
+  const pickImage = async () => {
+    let result = await ImagePicker.launchImageLibraryAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.Images,
+      allowsEditing: true,
+      aspect: [4, 3],
+      quality: 1,
+    });
+
+    console.log(result);
+
+    if (!result.cancelled) {
+      setImage(result.uri);
+    }
+  };
+
+  function sendValues(enteredimage, entereditemname, entereditemdesc, entereduseruni) {
+    if (!(enteredimage && entereditemname && entereditemdesc && entereduseruni)) {
+      throw new Error("Please do not leave any fields empty!");
+    } else if (enteredimage === UploadItemPhotoURI) {
+      throw new Error("Have you upload the item image?");
+    } else {
+      newProduct = new Product(entereditemname,auth.currentUser.uid,entereditemdesc,enteredimage);
+      
+      return new AuctionView(auth, db , newProduct)
+
+    }
+    
+};
     
+    const db = getFirestore();
+    getDoc(doc(db, "users", auth.currentUser.uid)).then(docSnap => {
+        if (docSnap.exists()) {
+          //console.log("Document data:", docSnap.data());
+          const data = docSnap.data();
+          //for (let [key, value] of Object.entries(data)) {
+            //console.log(`${key}: ${value}`);
+          //}
+          const originUni = data.originUni;
+          switch (originUni) {
+            case "NUS" :
+                // Do work here
+                //console.log('@u.nus.edu');
+                setUseruni('National University of Singapore (NUS)');
+                break;
+            case "NTU" :
+                // Do work here
+                //console.log('@e.ntu.edu.sg');
+                setUseruni('Nanyang Technological University (NTU)');
+                break;
+            case "SMU" :
+                // Do work here
+                //console.log('@smu.edu');
+                setUseruni('Singapore Management University (SMU)');
+                break;
+            case "SIT" :
+                // Do work here
+                //console.log('@sit.singaporetech.edu.sg');
+                setUseruni('Singapore Institute of Technology (SIT)');
+                break;
+            case "SUTD" :
+                // Do work here
+                //console.log('@sutd.edu.sg');
+                setUseruni('Singapore University of Technology & Design (SUTD)');
+                break;
+            case "SUSS" :
+                // Do work here
+                //console.log('@suss.edu.sg');
+                setUseruni('Singapore University of Social Sciences (SUSS)');
+                break;
+            case "GMAIL" :
```

Comment:
Similarly Here

Reply:
Noted, next merge in the development branch will be removed.